### DISCUSSION TITLE
Cisco Live 2022

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#9450] [General]
Anyone going to [Cisco Live US](https://www.ciscolive.com/) this year? @jordanrvillarreal and I will both be there to represent NS1 and talk about all things NetBox. If you'd like to schedule time to chat with either of us, please feel free to book a slot via Calendly ([Jeremy](https://calendly.com/jeremy-stretch/cisco-live-2022), [Jordan](https://calendly.com/jvillarreal-netbox/cisco-live-2022-meet-up)).

We were also thinking it'd be cool to put together an informal meetup somewhere around Mandalay Bay or the Luxor. Let us know if you're interested!

**Comment by kopacko** [comment]
I haven't been to a Live since COVID happened. But if I was going, I would 100% love to say hi.
